elements/urth-core-behaviors/jupyter-widget-behavior.html

```diff
@@ -6,6 +6,13 @@
 <!--
 This behavior is used to encapsulate some of the services needed for ipywidgets.
 -->
+<style>
+    .output_area pre.urth-widget-error {
+        color: red;
+        background-color: #fdd;
+        padding: 5px;
+    }
+</style>
```

Comment:
At first, seemed odd to have styling in a behavior. An alternative would be to use a `<style is="custom-style">` [shared style](https://www.polymer-project.org/1.0/docs/devguide/styling.html#style-modules) but thats probably a bit much for just this little bit of styling.